Makefile.vars.mk

```diff
@@ -18,7 +18,12 @@ KIND_KUBECONFIG ?= $(TESTBIN_DIR)/kind-kubeconfig-$(KIND_NODE_VERSION)
 KIND_CLUSTER ?= k8up-$(KIND_NODE_VERSION)
 KIND_KUBECTL_ARGS ?= --validate=true
 
-E2E_TAG ?= e2e_$(shell sha1sum $(BIN_FILENAME) | cut -b-8)
+ifeq ($(shell which sha1sum 2>&1; echo $$?),0)
```

Comment:
`which` should be avoided in favour of e.g. `command` or `hash` because it is not consistent among operating systems: https://stackoverflow.com/a/677212/172132

Reply:
Great point.